tmux-cc/src/tmux.pest

```diff
@@ -23,6 +24,7 @@ window_renamed = { "%window-renamed " ~ window_id ~ " " ~ any_text }
 session_changed = { "%session-changed " ~ session_id ~ " " ~ any_text }
 session_renamed = { "%session-renamed " ~ any_text }
 session_window_changed = { "%session-window-changed " ~ session_id ~ " " ~ window_id }
+layout_change = { "%layout-change " ~ window_id ~ " " ~ (window_layout ~ " " ~ window_layout ~ " " ~any_text | window_layout) }
```

Comment:
There's two types of `layout_change`. In tmux 2.x it is:

`%layout-change <window_id> <window_layout>`

in tmux 3.x it's

`%layout-change <window_id> <window_layout> <window_visible_layout> <window_raw_flags>`

I'm not quite sure the exact version where this new version is used. But I think the code should be compatible with both versions.

See https://github.com/tmux/tmux/blob/6c2bf0e22119804022c8038563b9865999f5a026/control-notify.c#L51